multilep/test/submitAll.py

```diff
@@ -8,7 +8,7 @@
 productionLabel = os.path.basename(datasetsFile.split('/')[-1].split('.')[0])                             # Label to keep track of the tuple version (is taken from the name of the above input file)
 datasets        = [dataset.strip() for dataset in open(datasetsFile)]                                     # Get list of datasets from file given as first argument
 datasets        = [dataset.split()[0] for dataset in datasets if dataset and not dataset.startswith('#')] # Clean empty and comment lines
-extraContent    = []
+extraContent    = ['']
```

Comment:
This change does not seem justified, because it will create extraContent strings starting with a comma

Reply:
Ah yes, this was a typo. I had removed the extracontent I had put there so other people didnt have it standard in their trees but I forgot to remove the ''